benchmark/application/coffee_pyomo.py

```diff
@@ -0,0 +1,85 @@
+shipping_cost_from_roastery_to_cafe = {
+    ('roastery1', 'cafe1'): 5,
+    ('roastery1', 'cafe2'): 3,
+    ('roastery1', 'cafe3'): 6,
+    ('roastery2', 'cafe1'): 4,
+    ('roastery2', 'cafe2'): 5,
+    ('roastery2', 'cafe3'): 2
+}
+light_coffee_needed_for_cafe = {'cafe1': 20, 'cafe2': 30, 'cafe3': 40}
+dark_coffee_needed_for_cafe = {'cafe1': 20, 'cafe2': 20, 'cafe3': 100}
+
+# Create a new model
+model = ConcreteModel()
+
+# Variables
+model.x = Var(shipping_cost_from_supplier_to_roastery.keys(),
+              domain=Integers, bounds=(0, None))
+model.y_light = Var(shipping_cost_from_roastery_to_cafe.keys(),
+                    domain=Integers, bounds=(0, None))
+model.y_dark = Var(shipping_cost_from_roastery_to_cafe.keys(),
+                   domain=Integers, bounds=(0, None))
+
+# Objective
+model.obj = Objective(expr=sum(model.x[i] * shipping_cost_from_supplier_to_roastery[i]
+                               for i in shipping_cost_from_supplier_to_roastery) +
+                      sum(roasting_cost_light[r] * model.y_light[r, c] +
+                          roasting_cost_dark[r] * model.y_dark[r, c]
+                          for r, c in shipping_cost_from_roastery_to_cafe) +
+                      sum((model.y_light[j] + model.y_dark[j]) * shipping_cost_from_roastery_to_cafe[j]
+                          for j in shipping_cost_from_roastery_to_cafe),
+                      sense=minimize)
+
+# Constraints
+
+
+def flow_constraint(model, r):
+    return sum(model.x[i] for i in shipping_cost_from_supplier_to_roastery if i[1] == r) == \
+        sum(model.y_light[j] + model.y_dark[j]
+            for j in shipping_cost_from_roastery_to_cafe if j[0] == r)
+
+
+def supply_constraint(model, s):
+    return sum(model.x[i] for i in shipping_cost_from_supplier_to_roastery if i[0] == s) <= capacity_in_supplier[s]
+
+
+def light_demand_constraint(model, c):
+    return sum(model.y_light[j] for j in shipping_cost_from_roastery_to_cafe if j[1] == c) >= light_coffee_needed_for_cafe[c]
+
+
+def dark_demand_constraint(model, c):
+    return sum(model.y_dark[j] for j in shipping_cost_from_roastery_to_cafe if j[1] == c) >= dark_coffee_needed_for_cafe[c]
+
+
+# model.constraints = ConstraintList()
+roasteries = list(set(i[1]
+                  for i in shipping_cost_from_supplier_to_roastery.keys()))
+suppliers = list(set(i[0]
+                 for i in shipping_cost_from_supplier_to_roastery.keys()))
+cafes = list(set(i[1] for i in shipping_cost_from_roastery_to_cafe.keys()))
+
+model.FlowConstraint = Constraint(roasteries, rule=flow_constraint)
+model.SupplyConstraint = Constraint(suppliers, rule=supply_constraint)
+model.LightDemandConstraint = Constraint(cafes, rule=light_demand_constraint)
+model.DarkDemandConstraint = Constraint(cafes, rule=dark_demand_constraint)
+
+# Solve
+# You can change the solver as per your requirement
+solver = SolverFactory('glpk')
+m = solver.solve(model)
+
+print('Optimal cost:', model.obj())
```

Comment:
Why remove the following?
```suggestion
print(m)

# OPTIGUIDE CONSTRAINT CODE GOES HERE

print(time.ctime())
if m.status == pyo.TerminationCondition.optimal:
    print(f'Optimal cost: {m.objVal}')
else:
    print("Not solved to optimality. Optimization status:", m.status)

```

Reply:
The purpose of `print(m)` seem not very valuable seen its containt information more than the needed (Problem, Solver, Solution). At now we just need status and objective value